tests/Fixer/FunctionNotation/MethodArgumentSpaceFixerTest.php

```diff
@@ -648,6 +648,39 @@ class FooClass
                 null,
                 ['ensure_fully_multiline' => true],
             ],
+            'test function argument with multiline echo in it' => [
+            <<<'UNAFFECTED'
+<?php
+call_user_func(function ($arguments) {
+    echo 'a',
+      'b';
+}, $argv);
+UNAFFECTED
```

Comment:
No it does not, since the original function call is not event partially multiline.